policyengine_it/parameters/gov/inps/social_security/rates/employee.yaml

```diff
@@ -0,0 +1,15 @@
+description: Social security rates for standard employees
+brackets:
+  - threshold:
+      values:
```

Comment:
we can just make this 0.4 no?

Reply:
I'm thinking of leaving this as it is. The "executive" employment category uses a `.calc()` function, which a standard parameter does not contain, and I think that manually switching based upon employment category when calculating would be less ideal.

Reply:
I would still make it a value parameter and removing the .calc function - this will also have positive frontend implications (less brackets / thresholds options which we are not dealing with optimally IMO)